Jenkinsfile_CNP

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
```

Comment:
you've added a new field in your ccd code called 
applicant1DQFixedRecoverableCostsIntermediate

Does it not need to be in caseData?

Reply:
For these tickets are purely creating the CCD screens and config.

applicant1DQFixedRecoverableCostsIntermediate
respondent1DQFixedRecoverableCostsIntermediate
respondent2DQFixedRecoverableCostsIntermediate

Will need to be added to casedata in service, when we want to use them in the Docmosis outputs, but to me it made more sense to do that in those tickets when they are picked up, rather than these CCD tickets

Reply:
I can update to add into casedata, will need done anyways, so either where it happens ticket wise.

Claim details wise, interesting point, no mention in tickets about that, makes sense though..